src/Controller/V2/Document/DocumentRoute.ts

```diff
@@ -159,19 +151,12 @@ export class DocumentRoute extends BaseRoute {
       payload,
       user
     )
-    res.sendStatus(StatusCodes.OK).end()
-    this.sendDataToClients(result, manuscriptID)
-  }
-  private async listen(req: Request, res: Response) {
-    const { manuscriptID, projectID } = req.params
-    const user = req.user
-    const result = await this.documentController.getEvents(projectID, manuscriptID, 0, user)
-    const data = this.formatDataForSSE({ ...result, transformVersion: getVersion() })
-    res.setHeader('Content-Type', 'text/event-stream')
-    res.setHeader('Connection', 'keep-alive')
-    res.setHeader('Cache-Control', 'no-cache')
-    res.write(data)
-    this.manageClientConnection(req, res)
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
I don't think we need this check, in the non-prod case, we just won't have a websocket to broadcast to.